### ISSUE TITLE
Priest shield break not working

### ISSUE THREAD

**Issue by slester** [AchaeaBashingScript/Bashing#30]
```
Your mind feels stronger and more alert.
5065h(100%) 3969m(100%) 96%e 92%w 14r edbE- (feyr118404 5%) 21:52:22.222 n|s (+540m, 13.6%) 
A nearly invisible magical shield forms around a hunchbacked feyr.
5065h(100%) 3969m(100%) 96%e 92%w 14r edbE- (feyr118404 5%) 21:52:22.527 n|s 

 |-- balance --|
 (2.392s)
5065h(100%) 3969m(100%) 96%e 92%w 14r exdbE- (feyr118404 5%) 21:52:23.873 n|s 
Brilliant! Dumb it down for me?
5065h(100%) 3969m(100%) 96%e 92%w 14r exdbE- (feyr118404 5%) 21:52:23.951 n|s 
You can use Torment again.
5065h(100%) 3969m(100%) 96%e 92%w 14r exdbE- (feyr118404 5%) 21:52:25.804 n|s 
You may drink another health or mana elixir. (4.518s)
5065h(100%) 3969m(100%) 96%e 92%w 14r exdbE- (feyr118404 5%) 21:52:26.740 n|s 
A dizzying beam of energy strikes you as your attack rebounds off of a hunchbacked feyr's shield.
5065h(100%) 3969m(100%) 96%e 92%w 14r xdbE- (feyr118404 5%) 21:52:26.801 n|s 
Grabbing you with tentacular appendages, a hunchbacked feyr's teeth sink into your flesh as its massive jaws clamp 
around you.
```

**Comment by slester** [comment]
And sometimes, I get this error instead:

`Non-numeric wildcard specified in command '%1'. Exiting alias.`

**Comment by keneanung** [comment]
Can you quote the output of `kconfig bashing`?

Which system do you use?

**Comment by slester** [comment]
svof, latest.

```
keneanung: Bashing is on
keneanung: Automatic fleeing is off
keneanung: Warning at a security threshhold of 500 health
keneanung: Fleeing at a security threshhold of 300 health
keneanung: Attack is set to smite
keneanung: Autoraze is on
keneanung: Special attack on shielding is set to none
keneanung: Razing shields with rage is on
keneanung: Currently using this battlerage strategy: simple
keneanung: Currently using this system: svo
keneanung: Considering loyals for fallback targets is off

keneanung: Loading these additional files on startup:    (Add new file)

keneanung: Version: 1.8-pre-b53517d-master
```

**Comment by keneanung** [comment]
`keneanung: Special attack on shielding is set to none`

This is the issue for the shield razing. You didn't set any command to use.

As for the other issue, I will open another bug where I can track info about that. You are not the first person telling me about this, but I can't remember who it was...

**Comment by slester** [comment]
is autoraze different from 'razing shields with rage'? I guess that's why.

**Comment by keneanung** [comment]
Autoraze is the balanceful action to raze shields like monk roundhouse combo or knight razeslash. I can't get the command for this one automatically, which is why the user has to set it.

Razing with rage uses the battlerage attack to raze shields. That one I can get automagically, because each class has that attack at the same position in battlerage (attack 3).
